src/components/LoadingView/LoadingView.js

```diff
@@ -0,0 +1,13 @@
+import { Background, Icon } from './LoadingView.styled'
+import azordevIcon from '../../assets/azordev-icon.png'
+
+
+const LoadingView = () => {
+	return (
```

Comment:
In this case you can
```suggestion
const LoadingView = () => (
```

Please check

Reply:
Thanks, that looks good